raiden/ui/cli.py

```diff
@@ -474,7 +474,9 @@ def smoketest(ctx, debug, local_matrix, **kwargs):  # pylint: disable=unused-arg
     )
 
     report_file = mktemp(suffix='.log')
-    configure_logging({'': 'DEBUG'}, log_file=report_file)
+    configure_logging(
+        {'': 'DEBUG'}, log_file=report_file,
```

Comment:
Can you add a keyword for the first parameter and put every parameter (and the final bracket on a new line.

Reply:
You're right, that looks better. Done in a053648.